### PR TITLE
[ShortCircuit] Corrections in javadoc

### PR DESCRIPTION
**Pull request by colinepiloquet**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

No

**What kind of change does this PR introduce?**

Docs update

**What is the current behavior?**

It is unclear when shortcircuit, short circuit and short-circuit is used.

**What is the new behavior (if this is a feature change)?**
I tried to make the English as correct as possible in the javadoc. ONLY modifications to the javadoc should be done.

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [x] No

### CODE REVIEW

File: shortcircuit-api/src/main/java/com/powsybl/shortcircuit/Fault.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 /**
  * Interface to describe the characteristics of the fault to be simulated.
- * Used for elementary short-circuit calculation only.
+ * Used for elementary short circuit analysis only.
```
Comment (So-Fras) [inline shortcircuit-api/src/main/java/com/powsybl/shortcircuit/Fault.java:23]:
```suggestion
 * Used for elementary short-circuit analysis only.
```

File: shortcircuit-api/src/main/java/com/powsybl/shortcircuit/BusFault.java [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 /**
  * Class to describe the characteristics of the fault to be simulated.
- * Used for elementary short-circuit calculation only.
+ * Used for elementary short circuit analysis only.
```
Comment (So-Fras) [inline shortcircuit-api/src/main/java/com/powsybl/shortcircuit/BusFault.java:11]:
```suggestion
 * Used for elementary short-circuit analysis only.
```

File: shortcircuit-api/src/main/java/com/powsybl/shortcircuit/BranchFault.java [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 /**
  * Class to describe the characteristics of the fault to be simulated.
- * Used for elementary short-circuit calculation only.
+ * Used for elementary short circuit analysis only.
```
Comment (So-Fras) [inline shortcircuit-api/src/main/java/com/powsybl/shortcircuit/BranchFault.java:11]:
```suggestion
 * Used for elementary short-circuit analysis only.
```

File: shortcircuit-api/src/main/java/com/powsybl/shortcircuit/AbstractFault.java [hunk 4]
```diff
@@ -10,7 +10,7 @@
 
 /**
  * Abstract class to describe the characteristics of the fault to be simulated.
- * Used for elementary short-circuit calculation only.
+ * Used for elementary short circuit analysis only.
```
Comment (So-Fras) [inline shortcircuit-api/src/main/java/com/powsybl/shortcircuit/AbstractFault.java:13]:
```suggestion
 * Used for elementary short-circuit analysis only.
```

File: shortcircuit-api/pom.xml [hunk 5]
```diff
@@ -20,7 +20,7 @@
 
     <artifactId>powsybl-shortcircuit-api</artifactId>
     <name>Short-circuit API</name>
-    <description>An API to run three phase short-circuit calculations</description>
+    <description>An API to run three-phase short circuit analysis</description>
```
Comment (So-Fras) [inline shortcircuit-api/pom.xml:23]:
```suggestion
    <description>An API to run three-phase short-circuit analysis</description>
```

File: powerfactory/powerfactory-converter/src/main/java/com/powsybl/powerfactory/converter/TransformerConverter.java [hunk 6]
```diff
@@ -344,7 +344,7 @@ private static TransformerModel aproximatePiModel(Complex z, Complex ym) {
         static Complex createImpedanceFromMeasures(double shortCircuitVoltage, double copperLosses,
             double ratedApparentPower, double nominalVoltage) {
 
-            // calculate leakage impedance from short circuit measures
+            // calculate leakage impedance from short-circuit measures
```
Comment (So-Fras) [inline powerfactory/powerfactory-converter/src/main/java/com/powsybl/powerfactory/converter/TransformerConverter.java:347]:
```suggestion
            // calculate leakage impedance from short-circuit measurements
```

**Review by So-Fras — CHANGES_REQUESTED** [review]
I suggested some changes regarding the short-circuit / short circuit use. Tell me if you agree!

As for measures/measurements, I am not 100% confident. In PowSyBl, we do use "measurement"...

**Comment by colinepiloquet** [comment]
> I suggested some changes regarding the short-circuit / short circuit use. Tell me if you agree!
> 
> As for measures/measurements, I am not 100% confident. In PowSyBl, we do use "measurement"...

I don't know either for measurements but I am fine with it. For some reason I had the feeling that with analysis "short circuit" was used but I looked a bit on IEC side and could not convince myself. To make it easy I would go with "short circuit" for a noun and "short-circuit' for a verb or adjective. So I followed your change requests. Thanks for reviewing that.

**Review by So-Fras — APPROVED** [review]

**Review by annetill — APPROVED** [review]

**Pull request merged**